app/bundles/LeadBundle/Config/config.php

```diff
@@ -502,6 +503,15 @@
                 'arguments' => ['mautic.factory'],
                 'alias'     => 'campaignevent_lead_field_value',
             ],
+            'mautic.form.type.campaignevent_lead_tags' => [
+                'class'     => 'Mautic\LeadBundle\Form\Type\CampaignEventLeadTagsType',
```

Comment:
This is very, very minor, but if you'd use 

`\Mautic\LeadBundle\Form\Type\CampaignEventLeadTagsType::class`

instead, it works the same way and your editor might help you with auto-complete when you type it.